### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by remonh87**

### CODE REVIEW

File: docs.json [hunk 1]
```diff
@@ -4,6 +4,10 @@
     "logo" : "https://user-images.githubusercontent.com/19904063/116995247-20519e80-acda-11eb-8e1b-7d0efbd193ad.png",
     "sidebar": [
         ["Overview", "/"],
+        ["Accept a Payment", [
+          ["Payment Sheet", "/sheet"]
```
Comment (remonh87) [inline docs.json:8]: @jamesblasco  how can I easily add the troubleshooting page + ideal page to it?

File: docs/troubleshooting.mdx [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+You need to make sure the minSdk in the build.gradle is specifified to 21 or above.
+
+```groovy
+defaultConfig {
+        applicationId "com.your.application"
+        minSdkVersion 21
+        targetSdkVersion 31
+```
+
+#### Kotlin version must be 1.5.0
```
Comment (jonasbark) [inline docs/troubleshooting.mdx:23]: must be at least*

File: docs/troubleshooting.mdx [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+Out of date Gradle dependencies can yield to weird build results and the App not to compile. Check that you are using the same version (or higher) as specifified in the example app. 
+- [Android tools](https://github.com/flutter-stripe/flutter_stripe/blob/main/example/android/build.gradle#L9) 
+- [Gradle version](https://github.com/flutter-stripe/flutter_stripe/blob/main/example/android/gradle/wrapper/gradle-wrapper.properties#L6) 
+
+#### Use FlutterFragmentActivity instead of `FluterActivity` as MainActivity
+
+The Stripe sdk requires a `FlutterFragmentActivity` instead of the `FlutterActivity` that is standard specifified when creating a new Flutter project. Follow the following steps to change it:
+
+1. Create a MainActivity kotlin file in the `Android` folder of your Flutter app. See the snippet below and our[example app](https://github.com/flutter-stripe/flutter_stripe/blob/79b201a2e9b827196d6a97bb41e1d0e526632a5a/example/android/app/src/main/kotlin/com/flutter/stripe/example/MainActivity.kt#L6).
```
Comment (jonasbark) [inline docs/troubleshooting.mdx:51]: I think the MainActivity class exists by default

Comment (jonasbark) [reply docs/troubleshooting.mdx:51]: also I think we should use main branch instead of the snapshot link

**Pull request merged**